lib/jekyll/entry_filter.rb

```diff
@@ -76,6 +71,10 @@ def excluded?(entry)
       end
     end
 
+    def excluded
```

Comment:
Presently, I don't see a need for this to be exposed as a public method.
So, please make it private.

Reply:
Done